content/working_with/console/pages/local-blueprints-page.md

```diff
@@ -1,22 +1,33 @@
 ---
 layout: bt_wiki
-title: Local Blueprints Page
+title: Blueprints Page
```

Comment:
Optional:

You can also change the file name from `local-blueprints-page.md` to `blueprints-page.md` and add alias using old page name.

Reply:
Done